src/modules/70-pipeline/components/PipelineDiagram/Nodes/DefaultNode/DefaultNode.module.scss

```
@@ -433,18 +471,32 @@
     }
   }
 }
+
+.disbaledIcon {
+  &:global(.bp3-icon) {
+    svg {
+      path:first-child {
+        fill: #d9dae5 !important;
```

Please use color variable
```suggestion
        fill: var(--grey-200) !important;
```